spiderfoot_neo4j/sfgraph.py

```diff
@@ -5,22 +5,19 @@
 import argparse
 from pathlib import Path
 
-# fscking imports amirite
```

Comment:
What did my sardonic comments do to deserve this

Reply:
😄 my apologies, sometimes I "professionalize" projects on autopilot without even processing the comedic value